cloud/pkg/admissioncontroller/config/kube.go

```diff
@@ -0,0 +1,27 @@
+package config
+
+import (
+	"github.com/kubeedge/beehive/pkg/common/config"
+	"github.com/kubeedge/beehive/pkg/common/log"
```

Comment:
please use klog

Reply:
klog should be a another feature that would be going to address, there should be some klog specific config needed to enable the klog, I'd prefer to not mixing klog with this PR chain.

Reply:
Klog have merged now, and no need specific config, and now have no beehive log in the repo, we can change to klog in this PR：）

Reply:
done